content/en/account_management/teams.md

```diff
@@ -0,0 +1,107 @@
+
+To associate a resource with a defined team, a Team must exist in Datadog with a matching team handle. When you click on a resource associated with a defined team, a small window appears with the team handle and additional information. Defined teams provide additional functionality such as the Team filter below. 
+
+Team handles that aren’t associated with a defined team in Datadog have a tag-like experience. Convert any undefined team handles to defined teams to take advantage of Teams features. 
+
+### Associate resources with team handles
+
+Datadog supports associating the following resources with team handles:
+
+- Dashboards
+- Service Catalog
+- Incidents
+- Monitors
+
+## Filter
+
+The team filter tailors user experiences to content associated with their team.
+
+The team filter appears in two places in each list view: 
+- A list of search facets at the top left
+- A search term in the search bar
+
+Enabling a team filter refines the list of resources based on a user's team memberships or the services owned by their teams. The team filter provides a global and persistent context that follows a user’s journey across products.
+
+The table below describes the products in which you can use the team filter:
+
+| Product List Page       | Filter basis                                                                     |
+|-------------------------|----------------------------------------------------------------------------------|
+| [Dashboards][2]         | Team handle                                                                      |
+| [Service Catalog][3]    | Team handle                                                                      |
+| [Incidents][4]          | Team handle                                                                      |
+| [Monitors][5]           | Team handle                                                                      |
+| [APM Error Tracking][6] | Service owned by teams (determined by ownership inside the [Service Catalog][3]) |
+| [Logs Error Tracking][7] | Service owned by teams (determined by ownership inside the [Service Catalog][3]) |
+
+
+## Permissions
+
+Any user in a role with the Teams Manage permission can create teams, rename teams, delete teams, and change team handles. Users with `user_access_manage` can add, remove, and promote team members and managers.
+
+### Team membership
+
+To differentiate members of your team, designate them as team managers. In the member list, team managers have a key icon next to their names. 
+
+Under the team's settings, specify which users can modify the team membership. The following options are available:
+- Only users with the `user_access_manage` permission
+- Team managers
+- Team managers and members
+- Anyone in the organization
+
+Users with the `user_access_manage` permission can set default rules on who can add or remove members, or edit team details. These policies can be overridden for individual teams.
```

Comment:
Maybe we say where they can set this? -  Something like "can set default rules on who can add or remove members, or edit team details via the Default Settings button on the Team List page

Reply:
Also tempted to add a small help section here along the lines of - 

Delegating Team Management
- If you prefer to have an open membership model, set your Default Team Settings to "Anyone in the Organization" can add or remove members, and assign to teams_manage permission to the appropriate roles to let anyone create teams or edit team details. 
- If you prefer a team driven membership model, set your Default Team Settings to "Team Managers" or "Team Managers and Members" can add or remove members, and assign to teams_manage permission to a role containing all your team managers. 
- If you prefer a strict membership model, set your Default Team Settings to "Only users with user_access_manage"  can add or remove members, and only assign the teams_manage permission to Org Admins.